config/bootstrap.php

```diff
@@ -41,7 +40,6 @@
 use Cake\Http\ServerRequest;
 use Cake\Log\Log;
 use Cake\Mailer\Email;
-use Cake\Utility\Inflector;
```

Comment:
for these lines 
https://github.com/cakephp/app/blob/aeca5e2a4c69999beaa743cd41286407fe3b37e3/config/bootstrap.php#L200-L203
we can not remove it

Reply:
Oh yea, thank you for the notice.
Actually, that code was commented, so let me add that back.

Reply:
Added the statement back to file.